portal_partner_select_all/wizard/portal_wizard.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+class PortalWizard(models.TransientModel):
+    _inherit = "portal.wizard"
+
+    set_all_user_ids = fields.Boolean(
```

Comment:
As a convention, fields finished with `_ids` are x2many. Since it is a boolean, I think it's better to call it `set_all_users`.